fastlane/lib/fastlane/actions/create_xcframework.rb

```diff
@@ -14,9 +14,15 @@ def self.run(params)
         create_command << ['-output', "\"#{params[:output]}\""]
         create_command << ['-allow-internal-distribution'] if params[:allow_internal_distribution]
 
+        if File.directory?(params[:output])
+          UI.message("Deleting existing: #{params[:output]}")
+          FileUtils.remove_dir(params[:output])
+        end
+
         Actions.lane_context[SharedValues::XCFRAMEWORK_PATH] = params[:output]
 
         sh(create_command)
+        UI.message("Successfully created: #{params[:output]}")
```

Comment:
That is right. I seem to have run tests on some other branch 🤦🤦🤦
I could just move this line up, but it is unnecessary, because the action prints anyway the absolute path where it was created 😅
So I removed this line.
Sorry and thanks 👍 👍